packs/data/extinction-curse-bestiary.db/zashathal-head-taker.json

```
@@ -759,7 +865,14 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "ActiveEffectLike",
+                        "mode": "override",
+                        "path": "flags.pf2e.rageTempHP",
+                        "value": 12
+                    }
+                ],
```

```suggestion
                "rules": [],
```
This should just be an effect, it's not related to their rage beyond having the rage trait.